x-pack/plugins/apm/server/lib/helpers/create_es_client/create_apm_event_client/index.test.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { contextServiceMock } from 'src/core/server/mocks';
+import supertest from 'supertest';
+import { createApmEventClient } from '.';
+import { createHttpServer } from '../../../../../../../../src/core/server/test_utils';
+
+describe('create_apm_event_client', () => {
```

Comment:
I'd name this `createApmEventClient` after the function, not after the module file name.